src/ensembl/src/content/app/browser/browser-region-editor/BrowserRegionEditor.tsx

```diff
@@ -150,7 +144,18 @@ export const BrowserRegionEditor = (props: BrowserRegionEditorProps) => {
     }
   };
 
-  const closeForm = () => {
+  const closeForm = (event: Event) => {
+    if (
+      !stickRef.current ||
+      !locationStartRef.current ||
+      !locationEndRef.current ||
+      stickRef.current.contains(event.target as HTMLElement) ||
+      locationStartRef.current.contains(event.target as HTMLElement) ||
+      locationEndRef.current.contains(event.target as HTMLElement)
```

Comment:
I wonder if the time has come to try to use optional chaining :-)

Reply:
I agree 😄 I will use them here.

Reply:
We'll need another babel plugin for that, I suppose...

Reply:
True, I am working on this and will create a PR with the babel plugin and probably update the other NPM packages too.

Reply:
@azangru There is now a PR for this: #205

Reply:
✅